manifests/s/Symflower/SymflowerCLI/20890/Symflower.SymflowerCLI.installer.yaml

```
@@ -0,0 +1,15 @@
+# Created using wingetcreate 0.5.0.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.1.0.schema.json
+
+PackageIdentifier: Symflower.SymflowerCLI
+PackageVersion: 20890
+Installers:
+- InstallerLocale: en-US
+  Architecture: x86

```

```suggestion
  Architecture: x64
```
Installer is `x86_64` which is meant to be run on x64 devices.